tests/NHSD.BuyingCatalogue.API.IntegrationTests/Steps/ClientApplicationTypes/NativeDesktopSectionSteps.cs

```diff
@@ -24,5 +24,13 @@ public async Task ThenTheSolutionNativeDesktopHardwareRequirementsSectionContain
             context.SelectToken($"{Token}.native-desktop-hardware-requirements.answers.{token}")
                 .ToString().Should().Be(value);
         }
+
+        [Then(@"the solution native-desktop native-desktop-connection-details section contains (.*) with value (.*)")]
```

Comment:
Could we restrict this to the specific question Id. My worry here is that it allows a developer to put something wrong in by "(.*)". Eventually we will need another step similar to this to ensure the specific question is not in the response body